app/uk/gov/hmrc/vulnerabilities/model/VulnerabilityCount.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vulnerabilities.model
+
+import play.api.libs.functional.syntax.{toFunctionalBuilderOps, unlift}
+import play.api.libs.json.{OFormat, __}
+
+case class VulnerabilityCount(
+  service       : String
+, environment   : String
+, curationStatus: CurationStatus
+, count         : Int
```

Comment:
CLARIFICATION: Would it make sense to change this to

    Map[CurationStatus, int]
    
and just have one VulnerabilityCount record per service/environment combination?